packages/blade/src/components/Input/BaseInput/BaseInput.stories.tsx

```diff
@@ -271,9 +316,9 @@ const UncontrolledBaseInputTemplate: ComponentStory<typeof BaseInputComponent> =
     />
   );
 };
-export const UncontrolledBaseInput = UncontrolledBaseInputTemplate.bind({});
+export const BaseInputUncontrolled = BaseInputUncontrolledTemplate.bind({});
 
-const ControlledBaseInputTemplate: ComponentStory<typeof BaseInputComponent> = () => {
+const BaseInputControlledTemplate: ComponentStory<typeof BaseInputComponent> = () => {
```

Comment:
On controlled example can we render the controlled value to a text element outside?

Reply:
what would be the benefit of that? since in the code snippet of input you're able to see the value prop being passed anyways? 🤔

Reply:
While I'm on the story can't see the code, atleast having the controls work might help.

Reply:
Okay let's keep it this way, It's a non-blocker.